src/main/kotlin/org/rust/cargo/project/configurable/CargoConfigurable.kt

```diff
@@ -17,12 +17,15 @@ import com.intellij.ui.dsl.builder.panel
 import com.intellij.ui.dsl.gridLayout.HorizontalAlign
 import org.rust.RsBundle
 import org.rust.cargo.project.model.isNewProjectModelImportEnabled
+import org.rust.cargo.project.settings.RustProjectSettingsService
+import org.rust.cargo.project.settings.rustSettings
 import java.awt.Component
 
 class CargoConfigurable(
     project: Project,
     private val isPlaceholder: Boolean
 ) : RsConfigurableBase(project, RsBundle.message("settings.rust.cargo.name")) {
+    private val settings: RustProjectSettingsService = project.rustSettings
```

Comment:
nit: maybe rename it somehow? Currently, the name is a bit confusing because it forces me to think it's a settings state, not a service